AnnoDesigner/Localization/LocalizeBinding.cs

```diff
@@ -51,4 +61,31 @@ public Localize(string key) : this()
             Key = key;
         }
     }
+
+    public class DynamicLocalize : MultiBinding
+    {
+        private static LocalizeConverter LocalizeConverter { get; } = new LocalizeConverter();
+
+        public string KeyPath
+        {
+            set
+            {
+                Bindings.Add(new Binding(value));
+            }
+        }
+
+        public DynamicLocalize()
```

Comment:
Can this be private if it should be never called, or is the public constructor required for it to be usable in XAML? Same question for the `LocalizeBinding` as well.

Reply:
Empty constructor is required if you want to use the `{l:DynamicLocalize KeyPath=Something}`